src/cli.ts

```diff
@@ -36,6 +36,11 @@ yargs.command("$0 <grammar> [output_file]", "Build parser from grammar",
                 default: false,
                 desc: "Use numeric enums for AST kinds",
             },
+            "union-enums": {
+                type: "boolean",
+                default: false,
+                desc: "Use unions instead of enums for AST kinds",
```

Comment:
Can you update the description also. Maybe include a link to https://www.typescriptlang.org/tsconfig/#erasableSyntaxOnly